### ISSUE TITLE
8 - Support for existing forms

### ISSUE THREAD

**Issue by worldspawn** [FineUploader/fine-uploader#781]
Hey. This seems like the minimum viable for this component but I can't see a way to do it. I would like to use the fineupload to select 1 or more files and just have append old fashioned input type=file elements to my form and then have my form be submitted normally (with a submit button).

I've read a few suggestions like using setParams to append my other form field data to the submission but that is.. yuck. Grossly complicating what should be a really basic operation.

In summary - i dont want the uploader component to be in any way responsible for submitting the files to the server. Is this supported or not?

**Comment by zhandosso** [comment]
Hi, @rnicholus

I am having trouble with form support feature:
I want to validate firstly other form fields, like text inputs, drop-down lists and only if all of them validated, call uploadStoredFiles(). But fine-uploader intercepts submit and tries to uploadStoredFiles(). 
My questions is: how can I cancel intercepting the submit. I have used option interceptSubmit: false, but it doesnt work. I have always "true" for interceptSubmit on creating form-support module:
      _ qq.FormSupport = function(options, startUpload, log) {
               "use strict";
            var self  = this,
_            _interceptSubmit = options.interceptSubmit_,_
            formEl = options.element,
            autoUpload = options.autoUpload;_

issue #2:
I want fine-uploader allow sending form data without file attachements. How could I disconnect(cancel) from fine-uploader once it is instantiated?

**Comment by zhandosso** [comment]
I am sorry, interceptSubmit = false solved. I realised that it must be passed as property of form:
form: {
    interceptSubmit = false
}
